content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2319,6 +2357,18 @@ The name of the `name` argument on the `@relation` attribute can be omitted (`re
 @relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?)
 ```
 
+As of 2.28.0, if the `referentialActions` preview flag is enabled the signature changes.
+
+```ts no-lines
+@relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?, onDelete: ReferentialAction?, onUpdate: ReferentialAction?, map: String?)
+```
+
+As of 2.29.0, if the `namedConstraints` preview flag is enabled the signature changes again unless you are using SQLite.
+
+```ts no-lines
+@relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?, onDelete: ReferentialAction?, onUpdate: ReferentialAction?, map: String?)
```

Comment:
I think this is copy/paster from the one just above and should probably be replace @do4gr

Reply:
yep, the top one does not have the map one yet

Reply:
changed it